launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -96,6 +96,14 @@
   <arg name="fusion_distance" default="100.0"/>
   <arg name="trust_object_distance" default="100.0"/>
 
+  <!-- Radar long range integration methods -->
+  <arg
+    name="radar_long_range_integration"
```

Comment:
[IMO]
This argument should change from string value to bool value.
String argument should be used for more than two choice, but I guess this parameter have only two mode even in the future.
If this parameter changes to bool value, I think the parameter name should change from `radar_long_range_integration` to `use_radar_tracking_fusion`.

Reply:
Make sense. I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/5269/commits/d5445dca3bbff854e35bc303026e652d69fd4ec7.